### DISCUSSION TITLE
why do not use docker ?

### DISCUSSION THREAD

**Discussion by ItaloCobains** [andyatkinson/rideshare#173] [Ideas]
.

**Comment by andyatkinson** [comment]
Hi there. For dependencies like PostgreSQL, or for everything?

**Reply by ItaloCobains** [reply]
Hi, for everything.

**Reply by andyatkinson** [reply]
Hi there. Docker is used in later chapters on replication. For the initial install instructions, my preference was for readers to install their Postgres instances directly on their OS. Thanks.
